new-games-to-db/main.py

```
@@ -3,26 +3,26 @@
 
 import argparse
-import bs4
-import firebase_admin
-import os
 import re
-import requests
 import sys
 import time
-
 from datetime import datetime, timedelta, timezone
+from pprint import pprint
```

Ruff (0.11.9)

`pprint.pprint` imported but unused

Remove unused import: `pprint.pprint`

(F401)